packages/web-frontend/src/components/View/utils.js

```diff
@@ -69,9 +69,9 @@ export function getExportViewWrapper({ type, viewType }) {
     default: {
       return () => (
         <div>
-          <PDFExportViewTitle>{`'${
-            type || viewType
-          }' type visual does not support PDF export`}</PDFExportViewTitle>
+          <PDFExportViewTitle>{`${
+            viewType || type
+          } visual PDF export is coming soon`}</PDFExportViewTitle>
```

Comment:
How soon I wonder..? 😉

Reply:
[RN-605](https://linear.app/bes/issue/RN-605/allow-picture-pdf-or-png-export-of-matrix-tables-and-static-visuals) It you want to prioritise, so it is really up to you :joy: